compiler/modules/front/tests/testConst.bal

```diff
@@ -63,4 +63,12 @@ function validConstExprs() returns map<ConstEvalTest> {
         m[t[0]] = t;
     }
     return m;
+}
+
+function fromFilename(string filename) returns err:Location {
```

Comment:
Should be locFromFilename or something like that.

Reply:
fixed